app/src/main/java/org/mozilla/focus/fragment/DownloadsFragment.java

```diff
@@ -76,7 +77,9 @@ public void onReceive(Context context, Intent intent) {
 
     @Override
     public void tryLoadMore() {
-        // TODO: 2017/9/26
+        if (!mDownloadListAdapter.isLoading()){
```

Comment:
Could be improved: This will send unnecessary load mores whenever were scrolling down in the last page.